pkg/cpio/cpio.go

```diff
@@ -84,6 +84,11 @@ func (w Writer) WriteRecord(rec Record) error {
 	return w.rw.WriteRecord(rec)
 }
 
+func (w Writer) WriteReproducibleRecord(rec Record) error {
```

Comment:
I don't think this should be part of the cpio library. This is something that should be part of the using library, some wrapper function that modifies records.

func reproducibleRecord(f cpio.Record) cpio.Record

Same with WriteReproducibleRecords below.

This library should be concerned only with writing exactly what's given in cpio.Record to the cpio file. If there's some reason that those values should be altered, it should happen at a higher level.